ci-operator/step-registry/optional-operators/ci/subscribe/optional-operators-ci-subscribe-commands.sh

```diff
@@ -100,6 +100,9 @@ metadata:
 spec:
   sourceType: grpc
   image: "$OO_INDEX"
+  updateStrategy:
+    registryPoll:
+      interval: 10m
```

Comment:
Why is this needed?

Reply:
This is for the operator upgrading test.

Reply:
Why does the operator upgrading test not work without this?

Reply:
This is the OLM design, here is the doc [catalog-polling](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/catalog-polling.md#catalog-polling).

Reply:
The scenario described in this document will never happen, so this does not seem to be useful for anything. The index image will not change during the duration of the test, so it does not need to be polled.

Reply:
Yeah, but if we don't set it, we couldn't test the operator's upgrading. Reduce the interval time? Change the `interval: 10m` to `interval: 3m`?
Anyway, I remove them first now. Will add it back when we have to use it.